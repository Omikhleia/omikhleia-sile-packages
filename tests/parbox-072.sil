\begin[class=book]{document}% Needs book class for footnotes
\neverindent
\nofolios
\script[src=packages/parbox]
\script[src=packages/lorem]
% Test case 072 = Parbox inception and footnotes cascading as in 011, but with
% minimized enabled.

\parbox[width=50%fw, strut=character, padding=5pt, border=0.5pt, minimize=true]{%
I am a parbox\footnote{Main parbox footnote.}

\parbox[width=50%fw, strut=character, padding=5pt, border=0.5pt, minimize=true]{%
I am an insider parbox\footnote{Insider parbox footnote.}}
}

\end{document}