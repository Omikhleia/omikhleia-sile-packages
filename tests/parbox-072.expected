Set paper size 	595.275597	841.8897729
Begin page
Draw line	49.4079	97.6592	117.0380	0.5000
Draw line	49.4079	139.7530	117.0380	0.5000
Draw line	49.4079	97.6592	0.5000	42.5938
Draw line	166.4459	97.6592	0.5000	42.5938
Mx 	54.4079
My 	110.8135
Set font 	Gentium Plus;10;400;;normal;;LTR
T	44 w=2.9785	(I)
Mx 	60.0341
T	68 80 w=12.6221	(am)
Mx 	75.3039
T	68 w=4.5898	(a)
Mx 	82.5415
T	83 68 85 69 82 91 w=28.9844	(parbox)
Mx 	111.5258
My 	107.6348
Set font 	Gentium Plus;6.81152;400;;normal;;LTR
T	20 w=3.1962	(1)
Draw line	54.4079	114.1573	107.0380	0.5000
Draw line	54.4079	134.7530	107.0380	0.5000
Draw line	54.4079	114.1573	0.5000	21.0957
Draw line	161.4459	114.1573	0.5000	21.0957
Mx 	59.4079
My 	127.3116
Set font 	Gentium Plus;10;400;;normal;;LTR
T	44 w=2.9785	(I)
Mx 	65.0290
T	68 80 w=12.6221	(am)
Mx 	80.2936
T	68 81 w=10.1074	(an)
Mx 	93.0436
T	76 81 86 76 71 72 85 w=28.5791	(insider)
Mx 	124.2653
T	83 68 85 69 82 91 w=28.9844	(parbox)
Mx 	153.2497
My 	124.1328
Set font 	Gentium Plus;6.81152;400;;normal;;LTR
T	21 w=3.1962	(2)
Mx 	49.4079
My 	687.8323
Set font 	Gentium Plus;9;400;;normal;;LTR
T	20 w=4.2231	(1)
Mx 	53.6310
T	17 w=2.0610	(.)
Mx 	73.6921
T	48 68 76 81 w=18.7910	(Main)
Mx 	94.8965
T	83 68 85 69 82 91 w=26.0859	(parbox)
Mx 	123.3960
T	73 82 82 87 81 82 87 72 w=31.6978	(footnote)
Mx 	155.0937
T	17 w=2.0610	(.)
Mx 	49.4079
My 	699.1848
T	21 w=4.2231	(2)
Mx 	53.6310
T	17 w=2.0610	(.)
Mx 	73.6921
T	44 81 86 76 71 72 85 w=25.9629	(Insider)
Mx 	102.0677
T	83 68 85 69 82 91 w=26.0859	(parbox)
Mx 	130.5664
T	73 82 82 87 81 82 87 72 w=31.6978	(footnote)
Mx 	162.2642
T	17 w=2.0610	(.)
End page
Finish
